.github/workflows/cd.yml

```diff
@@ -0,0 +1,18 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    branches-ignore: ["*"]
+    types:
+      - completed
+  push:
+    tags:
+      - "^v.*"
+
+permissions:
+  contents: read
+
+jobs:
+  plugin-cd:
+    uses: mattermost/actions-workflows/.github/workflows/plugin-cd.yml@main
```

Comment:
I think we should fixate the version here so community member using this template have to opt into updates.

Reply:
like a release tag?

Reply:
Yes